gosec/gosec_report.js

```diff
@@ -34,12 +14,12 @@ module.exports = (results, directory) => {
   let title, summary, annotations
   if (results && results.Issues.length !== 0) {
     title = config.issuesFoundResultTitle
-    summary = customSummary(results.Issues)
     annotations = results.Issues.map(issue => getAnnotation(issue, directory))
-  } else {
-    title = config.noIssuesResultTitle
-    summary = config.noIssuesResultSummary
+    summary = countIssueLevels(annotations)
+    return { title, summary, annotations }
```

Comment:
Returning different objects at different places in a function causes the same problem (the caller will get a bunch of undefined if the function returns fewer values than expected).
Here I think we should always return the same structure (title + summary + annotations) and pick sensible defaults for the values that are not needed (empty string, empty array, ...)

Reply:
Yes, you are right. Your suggestion makes more sense.